### ISSUE TITLE
Vision Light Tech - ACF125B0158IRM12 - S-Mount lens

### ISSUE THREAD

**Issue by Lawproto** [icub-tech-iit/cad-libraries#179]
| Description          | Field                                  |
|:---------------------|:---------------------------------------|
| Datasheet component  | [link or attachment ]                  |
| Project              | Eyes Mk4  |
| Alias Creo           | VLT_ACF125B0158IRM12              |
| IIT-Code             | [if available]                         |
| Applicant            | Lorenzo Protopapa, @Lawproto                 |
| Supervisor           | Nicolò Genesio, @Nicogene                 |
| Maintainer           | Marcello Savoldi, @marcellosavoldi     |
| Coworkers            | @icub-tech-iit/silo-mech               |

### Attachment:
https://www.visionlighttech.com/storage/16878/ACF125B0158IRM12.pdf

### Checklist:
Complete the checklist in order to proceed with the request.

- [x] Created the Alias following [the standard convention](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#commercial-components-coding-standard).
- [x] Added, if necessary, the source files (STEP, IGS, etc), the links/datasheets/drawings or other information used to create the library part.
- [x] File imported using the Creo [start part commercial](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#standard-part-for-commercial-components).
- [ ] Checked [mass properties](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#mass-properties-of-commercial-parts).
- [x] Checked if one [special case](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#special-cases-and-examples) applies to this component.
- [x] Converted the files with the [EDU-COM license](https://github.com/icub-tech-iit/cad-libraries/wiki/PTC-Creo-Guidelines#save-file-with-commercial-license).
- [ ] Created IIT code in [WINGST](http://wingst.icub.iit.local/) with manufacturer and at least one supplier.

**Comment by Lawproto** [comment]
I created the item following the datasheet.

![immagine](https://github.com/icub-tech-iit/cad-libraries/assets/84333301/56c78ea9-f104-4744-9a51-fd080366b6e4)

Not sure if the `LE` category is correct for this item. `COMM` and `ELTR` also seem possible to me.

**Comment by Lawproto** [comment]
For now I added the same mass of the Basler. To be checked when we receive the physical objects.

**Comment by marcellosavoldi** [comment]
> I created the item following the datasheet.
> 
> 
> Not sure if the `LE` category is correct for this item. `COMM` and `ELTR` also seem possible to me.

The LE category in CAD is not usually used.
Better to use another category... like COMM .
Honestly ELTR I would only use it for components to be used on electronic boards (to be used on CAD-Collaboration).
what do others think?

**Comment by Lawproto** [comment]
@marcellosavoldi `ELTR` or `COMM` makes a huge difference because it establishes whether the item is gonna be filtered from the mechanical BoM or not. I can see the reasoning beyond the `ELTR` since they are usually mounted on a board module.

For instance, the Basler camera (4k head) has been categorized as `ELTR`, so I am assuming it appears somewhere in the electronic BoM. Is that so?

![immagine](https://github.com/icub-tech-iit/cad-libraries/assets/84333301/ca9c8538-f637-424d-8af7-67ed71c825b6)
 
cc @maggia80

**Comment by maggia80** [comment]
@Lawproto, yes we are used to including the lens in the Logic Schematics. We will add them in the BoM of the head logic

**Comment by Lawproto** [comment]
@maggia80 so `ELTR` be it. I am assuming the related board modules follow the same category.

**Comment by maggia80** [comment]
yes!

**Comment by marcellosavoldi** [comment]
Perfect... as long as it is present in the electronic BOM, because Wingst the ELTR category skips it.
great!

**Comment by Lawproto** [comment]
And to quote Little Red Riding Hood's Huntsman before slaying the Big Bad Wolf: "Long have I sought you." Closing.

![Image](https://github.com/user-attachments/assets/780e7757-e8a5-4b14-b545-86385f76b112)
